arborist/auth_request_json.go

```diff
@@ -14,7 +14,34 @@ type AuthRequestJSON_User struct {
 	// The Policies field is optional, and if the request provides a token
 	// this gets filled in by the first stage of handling by the engine, using
 	// the Token field.
-	Policies []string `json:"policies,omitempty"`
+	Policies  []string `json:"policies,omitempty"`
+	Audiences []string `json:"aud,omitempty"`
+}
+
+func (requestJSON *AuthRequestJSON_User) UnmarshalJSON(data []byte) error {
```

Comment:
shouldn't this replace the UnmarshalJSON function on line 56?

Reply:
They're actually different methods, one for the `user` field of the auth request and one for the `request` field (AuthRequestJSON_User`` vs `AuthRequestJSON_Request`)